src/Kdyby/Doctrine/DI/OrmExtension.php

```diff
@@ -39,7 +39,7 @@ class OrmExtension extends Nette\DI\CompilerExtension
 	 * @var array
 	 */
 	public $managerDefaults = array(
-		'metadataCache' => 'default',
+		'metadataCache' => null,
 		'queryCache' => 'default',
```

Comment:
I'm trying the filesystem cache now and it creates really deep directory structures. For example:

```
/temp/cache/Doctrine.Validator/41/d3/9f/97/c4/89/2f/9d/cf/a3/1f/01/71/1c/fc/98/6b/d2/d5/2f/1f/79/a9/46/68/a3/9f/38/8f/b4/ed/b7/DoctrineNamespaceCacheKey[Kdyby_kdyby-validator-cache-validator_c4bbc638].doctrinecache.data
```

I really hate this behaviour. :-/